tagilmo/VereyaPython/timestamped_video_frame.py

```diff
@@ -83,4 +92,4 @@ def __init__(self, message: TimestampedUnsignedCharVector,
 
     @property
     def pixels(self):
-        return cv2.imdecode(np.frombuffer(self._pixels, dtype="uint8"), cv2.IMREAD_COLOR)
+        return cv2.cvtColor(cv2.flip(np.frombuffer(self._pixels, dtype="uint8").reshape((self.iheight, self.iwidth, self.ich))[:,:,:3],0), cv2.COLOR_RGB2BGR)
```

Comment:
i think we can use numpy.flip and remove BGR conversion - this way we can drop opencv dependency.

Reply:
i can't drop rgb2bgr since it is necessary. Numpy flip - ok.

Reply:
Done